snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/SnomedConceptConverter.java

```diff
@@ -311,6 +312,33 @@ private void expandRelationships(List<SnomedConcept> results, final Set<String>
 			concept.setRelationships(new SnomedRelationships(conceptRelationships, null, null, conceptRelationships.size(), conceptRelationships.size()));
 		}
 	}
+	
+	private void expandInboundRelationships(List<SnomedConcept> results, final Set<String> conceptIds) {
+		if (!expand().containsKey(SnomedConcept.Expand.INBOUND_RELATIONSHIPS)) {
+			return;
+		}
+		
+		final Options expandOptions = expand().get(SnomedConcept.Expand.INBOUND_RELATIONSHIPS, Options.class);
+		final int relationshipSearchLimit = getLimit(expandOptions);
+		
+		final SnomedRelationships inboundRelationships = SnomedRequests.prepareSearchRelationship()
```

Comment:
Please, add support for filtering inbound relationships by `typeId` and `sourceId`.